src/unicef_attachments/models.py

```diff
@@ -39,7 +38,7 @@ def generate_file_path(attachment, filename):
     return '/'.join(file_path)
 
 
-class FileType(OrderedModel, models.Model):
+class FileType(models.Model):
```

Comment:
why remove the ordering?

Reply:
Didn't see why this was in place, and wanted to reduce complexity.

Reply:
If I remember correctly it was once added so that file types can be listed based on a specific priority in the front end when selecting them... not sure if this is used anymore.

Reply:
It did not look like the ordering had been maintained, and so thought that it was not being used. Who should I check with to make sure?

Reply:
I'm not sure at this point, I think we should keep it for now until we can check everywhere, also this could be a useful feature, I know that BOs have asked in the past to have an ordering to their file types, i can see it potentially coming sometimes in the future as well.

Reply:
Sounds good, I'll revert these changes then